tests/integration/server.test.js

```diff
@@ -1,28 +1,14 @@
-'use strict';
-
-var request = require('supertest');
-var server = require('../../server');
+import request from 'supertest';
+import server from '../../server';
 
 var app;
 
-describe('server tests', () => {
-    beforeEach(() => {
-        app = server.start();
-    });
+describe('INTEGRATION-TESTS::Server tests', () => {
+    beforeEach(() => app = server.start());
 
-    afterEach((done) => {
-        app.close(done);
-    });
+    afterEach((done) => app.close(done));
```

Comment:
Code-style-wise, I like leaving off the parentheses around the `done` param there (single param, very unlikely to have a second param, etc).